propel/resource_data_pool_access_policy_test.go

```diff
@@ -0,0 +1,118 @@
+package propel
+
+import (
+	"context"
+	"testing"
+
+	"github.com/Khan/genqlient/graphql"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+
+	pc "github.com/propeldata/terraform-provider-propel/propel_client"
+)
+
+func TestAccPropelDataPoolAccessPolicyBasic(t *testing.T) {
```

Comment:
can you add a test where an application is assigned?, please

Reply:
Added an application to the test access policy :)